airflow/models/dagrun.py

```diff
@@ -307,7 +307,7 @@ def active_runs_of_dags(
         else:
             query = query.where(cls.state.in_((DagRunState.RUNNING, DagRunState.QUEUED)))
         query = query.group_by(cls.dag_id)
-        return {dag_id: count for dag_id, count in session.execute(query)}
+        return dict(session.execute(query))
```

Comment:
```suggestion
        return dict(iter(session.execute(query)))
```

I believe this is enought for Python to choose the correct dict constructor.